packages/grant-explorer/src/features/round/ThankYou.test.tsx

```diff
@@ -66,7 +66,7 @@ describe("createTwiterShareText", () => {
         isMrc: false,
       })
     ).toEqual(
-      `I just donated to a round on @gitcoin. Join me in making a difference by donating today, and check out the projects I supported on my Donation History page!
+      `I just donated to a round on @gitcoin @grantsstack. Join me in making a difference by donating today, and check out the projects I supported on my Donation History page!
```

Comment:
Should it be `@gitcoin's @grantsstack` (as in the text on https://twitter.com/grantsstack)?

Reply:
@michellema1208

Reply:
@bhargavaparoksham @bard good catch, let's do that!